pkg/ccl/changefeedccl/changefeed.go

```diff
@@ -46,32 +43,30 @@ func emitResolvedTimestamp(
 	return nil
 }
 
+func shouldProtectTimestamps(codec keys.SQLCodec) bool {
+	// TODO(smiskin): Remove this restriction once tenant based pts are enabled
```

Comment:
Can we TODO this @samiskin @adityamaru, given https://github.com/cockroachdb/cockroach/issues/73727#issuecomment-1061892910 ?